content/spin/quickstart.md

```diff
@@ -1,55 +1,145 @@
 title = "Taking Spin for a spin"
 template = "spin_main"
 date = "2022-03-14T00:22:56Z"
+enable_shortcodes = true
 [extra]
 url = "https://github.com/fermyon/spin/blob/main/docs/content/quickstart.md"
 keywords = "quickstart"
 
 ---
-- [Getting the `spin` Binary](#getting-the-spin-binary)
-- [Creating a New Spin Application From a Template](#creating-a-new-spin-application-from-a-template)
-- [Running the Application With `spin up`](#running-the-application-with-spin-up)
+- [Install Spin](#install-spin)
+- [Install the Prerequisites](#install-the-prerequisites)
+  - [Install a Template](#install-a-template)
+  - [Install the Tools](#install-the-tools)
+- [Create Your First Application](#create-your-first-application)
+- [Build Your Application](#build-your-application)
+- [Run Your Application](#run-your-application)
 - [Next Steps](#next-steps)
 
-> This is an early preview of the Spin project. It is still experimental code,
-> and you should expect breaking changes before the first stable release.
+Let's get Spin and take it from nothing to a "hello world" application!
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/sDiQV5RHorE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## Getting the `spin` Binary
+## Install Spin
 
-You can install the `spin` binary using the `install.sh` script hosted on this site:
+{{ tabs "os" }}
+
+{{ startTab "Linux"}}
+
+Download the `spin` binary using the `install.sh` script hosted on this site:
 
 <!-- @selectiveCpy -->
 
 <pre class="bash spin-install" id="spin-install-quick"><code>$ curl -fsSL https://developer.fermyon.com/downloads/install.sh | bash
 </code></pre>
 
-At this point, move the `spin` binary somewhere in your path, so it can be
-accessed from any directory. For example:
+Then move the `spin` binary somewhere in your path, so you can run it from anywhere. For example:
 
 <!-- @selectiveCpy -->
 
 ```bash
 $ sudo mv ./spin /usr/local/bin/spin
 ```
 
-More ways to [install Spin](install)
+{{ blockEnd }}
+
+{{ startTab "macOS"}}
+
+Download the `spin` binary using the `install.sh` script hosted on this site:
+
+<!-- @selectiveCpy -->
+
+<pre class="bash spin-install" id="spin-install-quick"><code>$ curl -fsSL https://developer.fermyon.com/downloads/install.sh | bash
+</code></pre>
+
+Then move the `spin` binary somewhere in your path, so you can run it from anywhere. For example:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ sudo mv ./spin /usr/local/bin/spin
+```
+
+{{ blockEnd }}
+
+{{ startTab "Windows"}}
+
+Download <a href="https://github.com/fermyon/spin/releases/latest" class="spin-install" id="spin-install-windows">the Windows binary release of Spin</a> from GitHub.
+
+Unzip the binary release and place the `spin.exe` in your system path.
+
+{{ blockEnd }}
+{{ blockEnd }}
+
+* [See more options for installing Spin](install)
```

Comment:
Could we just make this a regular paragraph and not a list.
```suggestion
[See more options for installing Spin](install)
```

Reply:
That's probably more consistent.  I kind of went back and forth on making things stand out vs making them fit in.  Thanks.